spanner/spansql/sql.go

```diff
@@ -254,6 +265,9 @@ func (cd ColumnDef) SQL() string {
 	if cd.NotNull {
 		str += " NOT NULL"
 	}
+	if cd.Default != nil {
```

Comment:
Both of `Default` and `Generated` are not nil, `(cd ColumnDef) SQL()` will generate invalid SQL like `... DEFAULT (...) AS (...) STORED`. I thought about adding a enum-like const that has Default, Generated or None as a value but I didn't implement that because it will break compatibility.